server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java

```diff
@@ -146,15 +146,6 @@ public void execute(String[] args) throws Exception {
         }
       }
 
-      // Remove the tracers, we don't use them anymore.
-      @SuppressWarnings("deprecation")
-      String path = siteConf.get(Property.TRACE_ZK_PATH);
-      try {
-        zapDirectory(zoo, path, opts);
```

Comment:
Do we need to remove this as a final step in the upgrade from 2.1 -> 3.0? I didn't look to see if this is already done.

Reply:
> Do we need to remove this as a final step in the upgrade from 2.1 -> 3.0? I didn't look to see if this is already done.

This should already be done in 2.1. We don't need to keep this for 3.0 upgrade, but we could suggest users remove the `/tracer` directory themselves, or run the ZooZap util with the property still set (which may be done already on `bin/accumulo-cluster stop` already.

Reply:
@ctubbsii - Is there a good place to log a warning for this? I don't see an Upgrader class yet for 3.0 so is that something we should create in another PR?

Reply:
I don't think we need a warning for this. I meant suggest it in the release notes. Accumulo no longer has to care about the existence of this directory. It should have attempted removal on upgrading to 2.1, and if it missed it, it's not a big deal.